README.md

```diff
@@ -11,25 +11,26 @@ ___
-    - [Using Revancify](https://github.com/inotia00/revanced-documentation/blob/main/docs/revancify.md#patching)
-  - [Method 3. RVX Builder (PC)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(pc).md)
+  - [Method 2. RVX Builder (PC)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(pc).md)
     - [Requirements](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(pc).md#requirements)
     - [Using RVX Builder (PC)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(pc).md#using-rvx-builder-windows--macos--linux)
-  - [Method 4. RVX Builder (Android - Termux)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md)
+  - [Method 3. RVX Builder (Android - Termux)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md)
     - [Installing RVX Builder (Android)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md#installing-rvx-builder-for-the-first-time)
     - [Using RVX Builder (Android)](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md#using-rvx-builder-termux)
       - [Additional info](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md#additional-info)
       - [Troubleshooting RVX Builder](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-builder%20(android).md#troubleshooting)
-  - [Method 5. ReVanced CLI (PC)](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md)
+  - [Method 4. ReVanced CLI (PC)](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md)
     - [Requirements](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md#requirements)
     - [Preparing the packages](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md#preparing-the-packages)
     - [Using ReVanced CLI](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md#using-revanced-cli-pc)
       - [Non-ROOT installations](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md#non-root-installations)
       - [ROOT installations](https://github.com/inotia00/revanced-documentation/blob/main/docs/revanced-cli.md#root-installations)
   - [Supplement. Patch for latest Reddit](https://github.com/inotia00/revanced-documentation/blob/main/docs/latest-reddit-patch-info.md)
 - [Information about the patches](https://github.com/inotia00/revanced-documentation/blob/main/docs/information-about-patches.md)
+
+> [!NOTE]
+> Patching with [Revancify](https://github.com/inotia00/revanced-documentation/blob/main/docs/revancify.md) is currently not possible because it uses patcher v21, which the ReVanced Extended patches are not yet compatible with.
```

Comment:
Revancify will support RVX within 24 hours
I will delete this sentence

Reply:
This entire commit should be reverted

https://github.com/inotia00/revanced-documentation/pull/9/commits/732067919126348b2358b6905845223ae9bd5f63